<!DOCTYPE html><html lang="en"><head><title>Publications</title><meta charset="utf-8"><!--[if IE]><meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'><![endif]--><meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1"><meta name="description" content=""><link href="https://fonts.googleapis.com/css?family=Karla:400,400i,700%7CLato:300,400,400i,700" rel="stylesheet"><link rel="stylesheet" href="css/bootstrap.min.css"><link rel="stylesheet" href="css/magnific-popup.css"><link rel="stylesheet" href="css/font-icons.css"><link rel="stylesheet" href="css/sliders.css"><link rel="stylesheet" href="css/style.css"><link rel="shortcut icon" href="img/favicon.ico"><link rel="apple-touch-icon" href="img/apple-touch-icon.png"><link rel="apple-touch-icon" sizes="72x72" href="img/apple-touch-icon-72x72.png"><link rel="apple-touch-icon" sizes="114x114" href="img/apple-touch-icon-114x114.png"></head><body><div class="loader-mask"><div class="loader">"Loading..."</div></div><main class="main-wrapper"><header class="nav nav--transparent"><div class="nav__holder" id="sticky-nav"><div class="container-fluid"><div class="flex-parent"><div class="nav__header flex-child clearfix"><div class="logo-wrap"><a class="logo__link"></a></div><button type="button" class="nav__icon-toggle" id="nav__icon-toggle" data-toggle="collapse" data-target="#navbar-collapse"><span class="sr-only">Toggle navigation</span> <span class="nav__icon-toggle-bar"></span> <span class="nav__icon-toggle-bar"></span> <span class="nav__icon-toggle-bar"></span></button></div><nav id="navbar-collapse" class="nav__wrap flex-child collapse navbar-collapse"><ul class="nav__menu"><li id="flex-child_first" class="flex-child active"><a href="index.html">Home</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li><a href="index.html">Home</a></li></ul></li><li class="flex-child"><a href="goal.html">Our<span class="newstitle_hide">/</span>Goal</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li><a href="goal.html">Goal</a></li></ul></li><li class="nav__dropdown flex-child"><a href="team.html">Team</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li class="team_nav">Principal Investigator</li><li><a href="team_Panchenko.html">Anna R. Panchenko</a></li><li class="team_nav">Research Associates</li><li><a href="team_Li.html">Shuxiang Li</a></li><li class="team_nav">Postdoctoral Fellows</li><li><a href="team_Yosef_Masoudi.html">Yosef Masoudi-Sobhanzadeh</a></li><li class="team_nav">Graduate Students</li><li><a href="team_Espiritu.html">Daniel Espiritu</a></li><li><a href="team_Ostroverkhova.html">Daria Ostroverkhova</a></li><li><a href="team_Rahman.html">Anisur Rahman</a></li><li><a href="team_Sheng.html">Yiru Sheng</a></li><li class="team_nav">Undergraduate Students</li><li><a href="team_Christos Kapetanios.html">Christos Kapetanios</a></li><a href="alumni.html"><li class="team_life">Alumni</li></a></ul></li><li class="nav__dropdown flex-child"><a href="project_3.html">Projects</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li><a href="project_3.html">Cancer Research</a></li><li><a href="project_1.html">Structural Bioinformatics</a></li><li><a href="project_2.html">Chromatin Dynamics</a></li></ul></li><li class="nav__dropdown flex-child"><a href="server.html">Servers/Databases</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li><a href="server_1.html">MutaGene</a></li><li><a href="server_2.html">HistoneDB</a></li><li><a href="server_3.html">HYDROID</a></li><li><a href="server_4.html">MutaBind</a></li><li><a href="server_6.html">Histone Interactome</a></li><li><a href="server_trj.html">Simulation Dataset</a></li></ul></li><li class="nav__dropdown flex-child"><a href="publications.html">Publications</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li><a href="publications.html">2024</a></li><li><a href="publications.html#2023">2023</a></li><li><a href="publications.html#2022">2022</a></li><li><a href="publications.html#2021">2021</a></li><li><a href="publications.html#2020">2020</a></li><li><a href="publications.html#2019">2019</a></li><li><a href="publications.html#2018">2018</a></li><li><a href="publications.html#2017">2017</a></li><li><a href="publications.html#2016">2016</a></li><li><a href="publications.html#2015">2015 and before</a></li></ul></li><li class="flex-child"><a href="lablife.html">Photo<span class="newstitle_hide">/</span>Gallery</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li><a href="lablife.html">Gallery</a></li></ul></li><li class="flex-child"><a href="news.html">News</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li><a href="news.html">News</a></li></ul></li><li class="flex-child"><a href="contact.html">Contact</a> <i class="ui-arrow-down nav__dropdown-trigger"></i><ul class="nav__dropdown-menu"><li><a href="contact.html">Contact</a></li></ul></li></ul></nav></div></div></div></header><div class="content-wrapper oh"><section class="page-title bg-img text-center" style="background: linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.3) ), url('../img/team/banner.jpg')"></section><section class="section-wrap pt-0"><div class="container"><h4 class="text-center">(current and former lab members are in bold)</h4><div class="heading-row text-center"><h2 class="heading">2024</h2></div><div class="row"><div class="col-xs-12"><ul><p id="2023"></p><li class="pub_text"><strong>Yosef Masoudi-Sobhanzadeh</strong>, <strong>Shuxiang Li</strong>, <strong>Yunhui Peng</strong>, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://www.biorxiv.org/content/10.1101/2024.02.09.579668v1" target="_blank">Genome-wide Nucleosome Positioning and Associated Features uncovered with Interpretable Deep Residual Networks</a>, biorxiv (2024).</li><li class="pub_text"><strong>Yunhui Peng</strong>, Wei Song, Vladimir Teif, Ivan Ovcharenko, David Landsman, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://elifesciences.org/reviewed-preprints/88936" target="_blank">Detection of new pioneer transcription factors as cell-type specific nucleosome binders</a>, eLife (2024).</li><li class="pub_text"><strong>Daria Ostroverkhova</strong>, Kathrin Tyryshkin, Annette K. Beach, Elizabeth A. Moore, <strong>Yosef Masoudi-Sobhanzadeh</strong>, Maria J Aristizabal, Stephanie R. Barbari, Igor B. Rogozin, Konstantin V. Shaitan, <strong>Anna R.Panchenko</strong>, Polina V. Shcherbakova. <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/38219146/" target="_blank">DNA polymerase ε and δ variants drive mutagenesis in polypurine tracts in human tumors</a>, Cell Reports (2024).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2023</h2></div><div class="row"><div class="col-xs-12"><ul><p id="2022"></p><li class="pub_text"><strong>Daria Ostroverkhova</strong>, <strong>Daniel Espiritu</strong>, Maria J Aristizabal, <strong>Anna R.Panchenko</strong>. <a class="pub_link" href="https://www.mdpi.com/2072-6694/15/13/3437" target="_blank">Leveraging Gene Redundancy to Find New Histone Drivers in Cancer</a>, Cancers (2023).</li><li class="pub_text">Maja Jensen, Venkataragavan Chandrasekaran, María-José García-Bonete, <strong>Shuxiang Li</strong>, Atsarina Anindya, Karin Andersson, Malin Erlandsson, Nina Oparina, Björn M. Burmann, Ulrika Brath, <strong>Anna R. Panchenko</strong>, Maria Bokarewa, Gergely Katona. <a class="pub_link" href="https://www.sciencedirect.com/science/article/pii/S2589004223010532" target="_blank">Survivin prevents the polycomb repressor complex 2 from methylating histone 3 lysine 27</a>, iScience (2023).</li><li class="pub_text"><strong>Daria Ostroverkhova</strong>, Teresa M Przytycka, <strong>Anna R.Panchenko</strong>. <a class="pub_link" href="https://www.sciencedirect.com/science/article/pii/S1471491423000679?via%3Dihub" target="_blank">Cancer driver mutations: predictions and reality</a>, Trends Mol Med (2023).</li><li class="pub_text"><strong>Shuxiang Li</strong>, <strong>Tiejun Wei</strong>, <strong>Anna R.Panchenko</strong>. <a class="pub_link" href="https://www.nature.com/articles/s41467-023-36465-5" target="_blank">Histone variant H2A.Z modulates nucleosome dynamics to promote DNA accessibility</a>, Nature Communications (2023).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2022</h2></div><div class="row"><div class="col-xs-12"><ul><p id="2021"></p><li class="pub_text">Seal, Ruth L., Paul Denny, Elspeth A. Bruford, Anna K. Gribkova, David Landsman, William F. Marzluff, Monica McAndrews, <strong>Anna R. Panchenko</strong>, <strong>Alexey K Shaytan</strong>, Paul B. Talbert. <a class="pub_link" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9526256/" target="_blank">A standardized nomenclature for mammalian histone genes</a>, Epigenetics & Chromatin (2022).</li><li class="pub_text"><strong>Shuxiang Li</strong>, <strong>Yunhui Peng</strong>, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://www.sciencedirect.com/science/article/pii/S0959440X22001099" target="_blank">DNA methylation: Precise modulation of chromatin structure and dynamics</a>, Curr Opin Struct Biol (2022).</li><li class="pub_text"><strong>Shuxiang Li</strong>, <strong>Yunhui Peng</strong>, David Landsman, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac097/6528903" target="_blank">DNA methylation cues in nucleosome geometry, stability, and unwrapping</a>, Nucleic Acids Research (2022).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2021</h2></div><div class="row"><div class="col-xs-12"><ul><li class="pub_text"><strong>Yunhui Peng</strong>, <strong>Shuxiang Li</strong>, Alexey Onufriev, David Landsman, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://www.nature.com/articles/s41467-021-25568-6.epdf?sharing_token=zHVNkIGyg2u0c56pFKC7xNRgN0jAjWel9jnR3ZoTv0PAp-fXwYyW7iDtECbgG5drvObHAHwqMh4ZqHdV_YLiaU4gLp7WRGsw-BO4k2mmW4TWApKCdNEkb4_v1_TBiIOpgMVd8Cw0YkKlbxkaumkAzcbHdecklAbd-XNCQ0S1E40%3D" target="_blank">Binding of regulatory proteins to nucleosomes is modulated by dynamic histone tails</a>, Nature Communications (2021).</li><li class="pub_text">Igor B. Rogozin, Abiel Roche-Lima, Kathrin Tyryshkin, Kelvin Carrasquillo-Carrión, Artem G. Lada, Lennard Y. Poliakov, Elena Schwartz, Andreu Saura, Vyacheslav Yurchenko, David N. Cooper, <strong>Anna R. Panchenko</strong>, Youri I. Pavlov <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/34093666/" target="_blank">DNA Methylation, Deamination, and Translesion Synthesis Combine to Generate Footprint Mutations in Cancer Driver Genes in B-Cell Derived Lymphomas and Other Cancers</a>, Front Genet (2021).</li><li class="pub_text">Yamini Dalal, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/33621519/" target="_blank">A Diving into Chromatin across Space and Time</a>, J Mol Biol (2021).</li><li class="pub_text"><strong>Daniel Espiritu</strong>, Anna K Gribkova, <strong>Shubhangi Gupta</strong>, <strong>Alexey K Shaytan</strong>, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://pubs.acs.org/doi/10.1021/acs.jpcb.1c00694" target="_blank">Molecular Mechanisms of Oncogenesis through the Lens of Nucleosomes and Histones</a> The Journal of Physical Chemistry B (2021)</li><li class="pub_text">Bing-Rui Zhou, Hanqiao Feng, <strong>Seyit Kale</strong>, Tara Fox, Htet Khant, Natalia de Val, Rodolfo Ghirlando, <strong>Anna R. Panchenko</strong>, Yawen Bai <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/33238161/" target="_blank">Distinct structures and dynamics of chromatosomes with different human linker histone isoforms</a> Mol Cell (2021).</li><li class="pub_text"><strong>Yunhui Peng</strong>, <strong>Shuxiang Li</strong>, David Landsman, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/33279866/" target="_blank">Histone tails as signaling antennas of chromatin</a>, Curr Opin Struct Biol (2021).</li><p id="2020"></p><li class="pub_text"><strong>Yunhui Peng</strong>, <strong>Yaroslav Markov</strong>, <strong>Alexander Goncearenco</strong>, David Landsman, <strong>Anna R.Panchenko</strong>. <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/33098859/" target="_blank">Human histone interaction networks: an old concept, new trends</a> J Mol Biol (2021).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2020</h2></div><div class="row"><div class="col-xs-12"><ul><li class="pub_text">Nitin Roper, <strong>Anna-Leigh Brown</strong>, Jun S. Wei, Svetlana Pack, Christopher Trindade, Chul Kim, Olivia Restifo, Shaojian Gao, Sivasish Sindiri, Farid Mehrabadi, Rajaa El Meskini, Zoe Weaver Ohler, Tapan K. Maity, Abhilash Venugopalan, Constance M. Cultraro, Elizabeth Akoth, Emerson Padiernos, Haobin Chen, Aparna Kesarwala, DeeDee K. Smart, Naris Nilubol, Arun Rajan, Zofia Piotrowska, Liqiang Xi, Mark Raffeld, <strong>Anna R. Panchenko</strong>, Cenk Sahinalp, Stephen Hewitt, Chuong D. Hoang, Javed Khan, Udayan Guha <a class="pub_link" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7263628/" target="_blank">Clonal Evolution and Heterogeneity of Osimertinib Acquired Resistance Mechanisms in EGFR Mutant Lung Cancer</a>, Cell Rep Medicine (2020).</li><li class="pub_text">Ning Zhang, Yuting Chen, Haoyu Lu Feiyang Zhao, Roberto Vera Alvarez, <strong>Alexander Goncearenco</strong>, <strong>Anna R. Panchenko</strong>, <strong>Minghui Li</strong> <a class="pub_link" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7068639/" target="_blank">MutaBind2: Predicting the Impacts of Single and Multiple Mutations on Protein-Protein Interactions</a>, iScience (2020).</li><p id="2019"></p><li class="pub_text">Jason A Somarelli, Heather Gardner, Vincent L Cannataro, Ella F Gunady, Amy M Boddy, Norman A Johnson, Jeffrey Nicholas Fisk, Stephen G Gaffney, Jeffrey H Chuang, Sheng Li, Francesca D Ciccarelli, <strong>Anna R Panchenko</strong>, Kate Megquier, Sudhir Kumar, Alex Dornburg, James DeGregori, Jeffrey P Townsend, <a class="pub_link" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6993850/" target="_blank">Molecular Biology and Evolution of Cancer: From Discovery to Action</a> Mol Biol Evol (2020).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2019</h2></div><div class="row"><div class="col-xs-12"><ul><li class="pub_text">Monika Fuxreiter, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/31337468/" target="_blank">Editorial overview: Dynamic protein interactions — from complexes to molecular machines</a>, Curr Opin Struct Biol (2019).</li><li class="pub_text">Arjan Hada, Swetansu K. Hota, Jie Luo, Yuan-chi Lin, <strong>Seyit Kale</strong>, <strong>Alexey K. Shaytan</strong>, Saurabh K. Bhardwaj, Jim Persinger, Jeff Ranish, <strong>Anna R. Panchenko</strong>, Blaine Bartholomew <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/31269447/" target="_blank">Histone octamer structure is altered early in ISW2 ATP-dependent remodeling of nucleosomes</a>, Cell Reports (2019).</li><li class="pub_text"><strong>Anna-Leigh Brown</strong>, Minghui Li, <strong>Alexander Goncearenco</strong>, <strong>Anna R Panchenko</strong>. <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/31034466/" target="_blank">Finding Driver Mutations in Cancer: Elucidating the Role of Background Mutational Processes</a>, PloS Comp Biol (2019).</li><li class="pub_text"><strong>Seyit Kale</strong>, <strong>Alexander Goncearenco</strong>, Yaroslav Markov, David Landsman, <strong>Anna R. Panchenko</strong>. <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/30991239/" target="_blank">Molecular recognition of nucleosomes by binding partners</a>, Curr Opin Struct Biol (2019).</li><p id="2018"></p><li class="pub_text">Nitin Roper, Shaojian Gao, Tapan K. Maity, A. Rouf Banday, Xu Zhang, Abhilash Venugopalan, Constance M. Cultraro, Rajesh Patidar, Sivasish Sindiri,Anna-Leigh Brown, <strong>Alexandr Goncearenco</strong>, <strong>Anna R. Panchenko</strong>, Romi Biswas, Anish Thomas, Arun Rajan, Corey A. Carter, David Kleiner, Stephen Hewitt, Javed Khan, Ludmila Prokunina-Olsson, Udayan Guha <a class="pub_link" href="https://pubmed.ncbi.nlm.nih.gov/30840888/" target="_blank">APOBEC Mutagenesis and Copy-Number Alterations Are Drivers of Proteogenomic Tumor Evolution and Heterogeneity in Metastatic Thoracic Tumors</a> Cell Reports (2019).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2018</h2></div><div class="row"><div class="col-xs-12"><ul><li class="pub_text"><strong>Alexey K. Shaytan</strong>, Hua Xiao, Grigoriy A. Armeev, Daria A. Gaykalova, Galina A. Komarova, Carl Wu, Vasily M. Studitsky, David Landsman, <strong>Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/30341436">Structural interpretation of DNA-protein hydroxyl-radical footprinting experiments with high resolution using HYDROID</a>, Nature Protocols (2018).</li><li class="pub_text">Feiyang Zhao, Lei Zheng, <strong>Alexander Goncearenco, Anna R. Panchenko and Minghui Li</strong> <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/30037003">Computational Approaches to Prioritize Cancer Driver Missense Mutations</a> Int J Mol Sci. (2018).</li><p id="2017"></p><li class="pub_text">Igor B. Rogozin, <strong>Alexander Goncearenco</strong>, Artem G. Lada, Subhajyoti De, Vyacheslav Yurchenko, German Nudelman, <strong>Anna R. Panchenko</strong>, David N. Cooper, Youri I. Pavlov <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/29139326">DNA polymerase ? mutational signatures are found in a variety of different types of cancer</a> Cell Cycle (2018).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2017</h2></div><div class="row"><div class="col-xs-12"><ul><li class="pub_text">Hua Xiao, Feng Wang, Jan Wisniewski, <strong>Alexey K. Shaytan</strong>, Rodolfo Ghirlando, Peter C. FitzGerald, Yingzi Huang, Debbie Wei, Shipeng Li, David Landsman, <strong>Anna R. Panchenko</strong> and Carl Wu <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/29074736">Molecular basis of CENP-C association with the CENP-A nucleosome at yeast centromeres</a> Genes Dev (2017).</li><li class="pub_text"><strong>Alexander Goncearenco, Minghui Li, Franco L. Simonetti</strong>, Benjamin A. Shoemaker and <strong>Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/28809006">Exploring Protein-Protein Interactions as Drug Targets for Anti-cancer Therapy with In Silico Workflows</a> Methods Mol Biol (2017).</li><li class="pub_text"><strong>Alexey K. Shaytan</strong>, Hua Xiao, Grigoriy A. Armeev, Carl Wu, David Landsman and <strong>Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://academic.oup.com/nar/article/doi/10.1093/nar/gkx616/3976474/">Hydroxyl-radical footprinting combined with molecular modeling identifies unique features of DNA conformation and nucleosome positioning</a> Nucleic Acids Res (2017).</li><li class="pub_text">Igor B. Rogozin, Youri I. Pavlov, <strong>Alexander Goncearenco</strong>, Subhajyoti De, Artem G. Lada, Eugenia Poliakov, <strong>Anna R. Panchenko</strong> and David N. Cooper <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/28498882">Mutational signatures and mutable motifs in cancer genomes</a> Brief Bioinform (2017).</li><li class="pub_text"><strong>Alexander Goncearenco, Stephanie L. Rager, Minghui Li</strong>, Qing-Xiang Sang, Igor B. Rogozin and <strong>Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/28472504">Exploring background mutational processes to decipher cancer genetic heterogeneity</a> Nucleic Acids Res (2017).</li><li class="pub_text"><strong>Minghui Li, Alexander Goncearenco and Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/28188534">Annotating Mutational Effects on Proteins and Protein Interactions: Designing Novel and Revisiting Existing Protocols</a> Methods Mol Biol (2017).</li><p id="2016"></p><li class="pub_text">Sara El Kennani, Annie Adrait, <strong>Alexey K. Shaytan</strong>, Saadi Khochbin, Christophe Bruley, <strong>Anna R. Panchenko</strong>, David Landsman, Delphine Pflieger and Jrme Govin <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/28096900">MS_HistoneDB, a manually curated resource for proteomic analysis of human and mouse histones</a> Epigenetics Chromatin (2017).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2016</h2></div><div class="row"><div class="col-xs-12"><ul><li class="pub_text">Igor B. Rogozin, Artem G. Lada, <strong>Alexander Goncearenco</strong>, Michael R.Green, Subhajyoti De, German Nudelman, <strong>Anna R. Panchenko</strong>, Eugene V. Koonin and Youri I. Pavlov <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27924834">Activation induced deaminase mutational signature overlaps with CpG methylation sites in follicular lymphoma and other cancers</a> Sci Rep (2016).</li><li class="pub_text">Romi Biswas, Shaojian Gao, Constance M. Cultraro, Tapan K. Maity, Abhilash Venugopalan, Zied Abdullaev, <strong>Alexey K. Shaytan</strong>, Corey A. Carter, Anish Thomas, Arun Rajan, Young Song, Stephanie Pitts, Kevin Chen, Sara Bass, Joseph Boland, Ken-Ichi Hanada, Jinqiu Chen, Paul S. Meltzer, <strong>Anna R. Panchenko</strong>, James C. Yang, Svetlana Pack, Giuseppe Giaccone, David S. Schrump, Javed Khan, and Udayan Guha <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27900369">Genomic profiling of multiple sequentially acquired tumor metastatic sites from an "exceptional responder" lung adenocarcinoma patient reveals extensive genomic heterogeneity and novel somatic variants driving treatment response</a> Mol Case Stud (2016).</li><li class="pub_text"><strong>Minghui Li, Franco L. Simonetti, Alexander Goncearenco and Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27150810">MutaBind estimates and interprets the effects of sequence variants on proteinprotein interactions</a> Nucleic Acids Research (2016)</li><li class="pub_text"><strong>Eli J. Draizen, Alexey K. Shaytan</strong>, Leonardo Mario-Ramrez, Paul B. Talbert, David Landsman and <strong>Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26989147">HistoneDB 2.0: a histone database with variantsan integrated resource to explore histones and their variants</a> Database(Oxford) (2016).</li><li class="pub_text">Richa Agarwala et al <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26615191">Database resources of the National Center for Biotechnology Information</a> Nucleic Acids Res (2016).</li><li class="pub_text"><strong>Alexey K. Shaytan</strong>, Grigoriy A. Armeev, <strong>Alexander Goncearenco</strong>, Victor B. Zhurkin, David Landsman and <strong>Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26699921">Coupling between histone conformations and DNA geometry in nucleosomes on a microsecond timescale: atomistic insights into nucleosome functions</a> J Mol Biol (2016).</li><p id="2015"></p><li class="pub_text"><strong>Minghui Li</strong>, Stephen C. Kales, Ke Ma, Benjamin A. Shoemaker, Juan Crespo-Barreto, Andrew L. Cangelosi, Stanley Lipkowitz and <strong>Anna R. Panchenko</strong>. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26676746">Balancing protein stability-activity tradeoff in cancer: approach for identifying driver mutations affecting CBL ubiquitin ligase activation</a> Cancer Research (2016).</li></ul></div></div><div class="heading-row text-center"><h2 class="heading">2015 and before</h2></div><div class="row"><div class="col-xs-12"><ul><li class="pub_text">Alexandr Goncearenco, Alexey Shaytan, Benjamin Shoemaker and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26213149">Structural Perspectives on the Evolutionary Expansion of Unique Protein-Protein Binding Sites, </a>Biophysical J (2015).</li><li class="pub_text">Anna R. Panchenko and Madan M. Babu <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26199202">Editorial overview: Linking protein sequence and structural changes to function in the era of next-generation sequencing,</a> Curr Opin Struct Biol (2015).</li><li class="pub_text">Alexey K. Shaytan, David Landsman and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/25731851">Nucleosome adaptability conferred by sequence and structural variations in histone H2A-H2B dimers,</a> Curr Opin Struct Biol (2015).</li><li class="pub_text">Richa Agarwala et al. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/25398906">Database resources of the National Center for Biotechnology Information</a>, Nucleic Acids Res (2015).</li><p id="2014"></p><li class="pub_text">Hafumi Nishi, Emek Demir and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/25451034">Crosstalk between signaling pathways provided by single and multiple protein phosphorylation sites,</a> J Mol Biol (2015).</li><li class="pub_text">Hafumi Nishi, Alexey Shaytan and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/25147561">Physicochemical mechanisms on protein regulation by phosphorylation,</a> Frontiers in Genetics (2014).</li><li class="pub_text">Alexandr Goncearenco, Benjamin Shoemaker, Dachuan Zhang, Alex Sarychev and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24931138">Coverage of protein domain families with structural protein-protein interactions: current progress and future trends</a>, Prog Biophys Mol Biol (2014).</li><li class="pub_text">Minghui Li, Marharyta Petukh, Emil Alexov and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://pubs.acs.org/doi/10.1021/ct401022c">Predicting the Impact of Missense Mutations on ProteinProtein Binding Affinity</a>, J Chem Theory Comput (2014).</li><p id="2013"></p><li class="pub_text">Acland A et al <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24259429">Database resources of the National Center for Biotechnology Information</a>, Nucleic Acids Res (2014).</li><li class="pub_text">Benjamin Shoemaker, Stefan Wuchty and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/24510594/">Computational Large-Scale Mapping of Protein-Protein Interactions Using Structural Complexes</a>, Curr Prot Prot Sci (2013).</li><li class="pub_text">Shannon Stefl, Hafumi Nishi, Marharyta Petukh, Anna R. Panchenko and Emil Alexov <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/23871686">Molecular Mechanisms of Disease-Causing Missense Mutations</a>, J Mol Biol (2013).</li><li class="pub_text">Hafumi Nishi, Manoj Tyagi, Shaolei Teng, Benjamin A. Shoemaker, Kosuke Hashimoto, Emil Alexov, Stefan Wuchty and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/23799087">Cancer missense mutations alter binding properties of proteins and their interaction networks</a>, Plos One (2013).</li><li class="pub_text">Minghui Li, Benjamin A. Shoemaker, Ratna R. Thangudu, Joan D. Ferraris, Maurice B. Burg and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/23734591">Mutations in DNA-Binding Loop of NFAT5 Transcription Factor Produce Unique Outcomes on ProteinDNA Binding and Dynamics</a>, J Phys Chem B (2013).</li><li class="pub_text">Hafumi Nishi, Kosuke Hashimoto, Thomas Madej and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/23663963/">Evolutionary, Physicochemical,and Functional Mechanisms ofProtein Homooligomerization</a>, Prog Mol Biol Transl Sci (2013).</li><li class="pub_text">Hafumi Nishi, Jessica H. Fong, Christiana Chang, Sarah A. Teichmann and Anna R. Panchenko. <a class="pub_link" target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/23364837/">Regulation of proteinprotein binding by coupling between phosphorylation and intrinsic disorder: analysis of human protein complexes</a>, Mol BioSyst (2013).</li><li class="pub_text">Acland A et al <a class="pub_link" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/23193264">Database resources of the National Center for Biotechnology Information</a>, Nucleic Acids Res (2013).</li><li class="pub_text">Kosuke Hashimoto, Igor B. Rogozin and Anna R. Panchenko. Oncogenic potential is related to activating effect of cancer single and double somatic mutations in receptor tyrosine kinases, Hum Mutat (2012).</li><li class="pub_text">Chakraborty A, Mandloi S, Lanczycki CJ, Panchenko AR, Chakrabarti S SPEER-SERVER: a web server for prediction of protein specificity determining sites, Nucleic Acids Res (2012).</li><li class="pub_text">Sayers EW et al. Database resources of the National Center for Biotechnology Information, Nucleic Acids Res (2012).</li><li class="pub_text">Benjamin A. Shoemaker , Dachuan Zhang, Manoj Tyagi, Rajesh R. Thangudu , Jessica H. Fong , Aron Marchler-Bauer , Stephen H. Bryant , Thomas Madej and Anna R. Panchenko. IBIS (Inferred Biomolecular Interaction Server) reports, predicts and integrates multiple types of conserved interactions for proteins, Nucleic Acids Res (2012).</li><li class="pub_text">Thomas Madej, Kenneth J. Addess, Jessica H. Fong, Lewis Y. Geer, Renata C. Geer, Christopher J. Lanczycki, Chunlei Liu, Shennan Lu, Aron Marchler-Bauer, Anna R. Panchenko, Jie Chen, Paul A. Thiessen, Yanli Wang, Dachuan Zhang and Stephen H. Bryant MMDB: 3D structures and macromolecular interactions, Nucleic Acids Res (2012).</li><li class="pub_text">Manoj Tyagi, Kosuke Hashimoto, Benjamin Shoemaker, Stefan Wuchty and Anna R. Panchenko. Large-scale mapping of human protein interactome using structural complexes, EMBO Reports (2012).</li><li class="pub_text">Manoj Tyagi, Ratna Thangudu, Dachuan Zhang, Stephen Bryant, Thomas Madej and Anna R. Panchenko. Homology Inference of Protein-Protein Interactions via Conserved Binding Sites, Plos One (2012).</li><li class="pub_text">Ratna R Thangudu, Stephen Bryant; Anna R Panchenko and Thomas Madej Modulating protein-protein interactions with small molecules: the importance of binding hotspots, J Mol Biol (2012).</li><li class="pub_text">Jessica H. Fong, Benjamin A. Shoemaker and Anna R. Panchenko. Intrinsic protein disorder in human pathways, Mol Biosyst (2012).</li><li class="pub_text">Hafumi Nishi, Kosuke Hashimoto and Anna R. Panchenko. Phosphorylation in protein-protein binding: effect on stability and function, Structure (2011).</li><li class="pub_text">Kosuke Hashimoto, Hafumi Nishi, Stephen Bryant and Anna R. Panchenko. Caught in self-interaction: evolutionary and functional mechanisms of protein homooligomerization, Phys Biol (2011).</li><li class="pub_text">Ruth Nussinov, Anna R. Panchenko and Teresa Przytycka Physics approaches to protein interactions and gene regulation, Phys Biol (2011).</li><li class="pub_text">Sayers EW et al. Database resources of the National Center for Biotechnology Information, Nucleic Acids Res (2011).</li><li class="pub_text">Inbar Cohen-Gihon, Jessica H. Fong, Roded Sharan, Ruth Nussinov, Teresa Przytycka and Anna R. Panchenko. Evolution of domain promiscuity in eukaryotic genomes-a perspective from the inferred ancestral domain architectures, Mol Biosys (2010).</li><li class="pub_text">Kosuke Hashimoto and Anna R. Panchenko. Mechanisms of protein oligomerization: the critical role of insertions and deletions in maintaining different oligomeric states, Proc Natl Acad Sci (2010).</li><li class="pub_text">Ratna Thangudu, Manoj Tyagi, Benjamin Shoemaker, Stephen Bryant, Anna R. Panchenko and Thomas Madej Knowledge-based annotation of small molecule binding sites in proteins, BMC Bioinformatics (2010).</li><li class="pub_text">Jessica Fong and Anna R. Panchenko. "Intrinsic disorder and protein multibinding in domain, terminal, and linker regions", Mol Biosys, Oct 1;6(10):1821-8 (2010).</li><li class="pub_text">Kosuke Hashimoto, Thomas Madej, Stephen Bryant and Anna R. Panchenko. "Functional states of homooligomers: insights from the evolution of glycosyltransferases" J Mol Biol., 399(1):196-206 (2010).</li><li class="pub_text">Saikat Chakrabarti and Anna R. Panchenko. "Structural and functional roles of coevolved sites in proteins", PloS One, Jan 6;5(1):e8591 (2010).</li><li class="pub_text">Sayers EW et al. "Database resources of the National Center for Biotechnology Information", Nucleic Acids Res, D5-16 (2010).</li><li class="pub_text">Judith E. Dayhoff, Benjamin A. Shoemaker, Stephen Bryant and Anna R. Panchenko. "Evolution of protein binding modes in homooligomers", J Mol Biol, v. 395(4), p. 860 (2010).</li><li class="pub_text">Michael Lobanov, Benjamin A. Shoemaker,Sergei O. Garbuzynskiy, Jessica H. Fong, Anna R. Panchenko and Oxana V. Galzitskaya "ComSin: database of protein structures in bound (complex) and unbound (single) states in relation to their intrinsic disorder", Nucleic Acids Res., D283-7 (2010).</li><li class="pub_text">Benjamin A. Shoemaker, Dachuan Zhang, Ratna R. Thangudu, Manoj Tyagi, Jessica H. Fong, Aron Marchler-Bauer, Stephen H. Bryant, Thomas Madej and Anna R. Panchenko "Inferred Biomolecular Interaction Server (IBIS) a web server to analyze and predict protein interacting partners and binding sites", Nucleic Acids Res. D518-24 (2010).</li><li class="pub_text">Manoj Tyagi, Benjamin A. Shoemaker, Stephen Bryant and Anna R. Panchenko. "Exploring functional roles of multibinding protein interfaces", Protein Science, v. 18(8), p.1674 (2009).</li><li class="pub_text">S. Teng S, Thomas Madej, Anna R. Panchenko and Emil Alexov "Modeling effects of human single nucleotide polymorphisms on protein-protein interactions", Biophys J, v.96(11) (2009).</li><li class="pub_text">Saikat Chakrabarti and Anna R. Panchenko. "Ensemble approach to predict specificity determinants: benchmarking and validation", BMC Bioinformatics, 10:207 (2009).</li><li class="pub_text">Jessica H. Fong, Benjamin A. Shoemaker, Sergey O. Garbuzynskiy, Michael Y. Lobanov, Oxana V. Galzitskaya and Anna R. Panchenko. "Intrinsic disorder in protein interactions: insights from a comprehensive structural analysis", PLoS Comput Biol, v.5(3) (2009).</li><li class="pub_text">Maricel G. Kann MG, Benjamin A. Shoemaker, Anna R. Panchenko and Teresa M. Przytycka "Correlated evolution of interacting proteins: looking behind the mirrortree", J Mol Biol, v.385, p.91 (2009).</li><li class="pub_text">Saikat Chakrabarti and Anna R. Panchenko "Coevolution in defining the functional specificity", Proteins, v.75, p.231 (2009).</li><li class="pub_text">Saikat Chakrabarti, Stephen Bryant and Anna R. Panchenko "Functional specificity lies within the properties and evolutionary changes of amino acids", J Mol Biol, v.373(3), p.801 (2007).</li><li class="pub_text">Benjamin A. Shoemaker and Anna R. Panchenko. "Deciphering protein- protein interactions. I. Experimental techniques and databases", PLoS Comput Biol, v. 3(4) (2007).</li><li class="pub_text">Thomas Madej, Anna R. Panchenko, Jie Chen and Stephen Bryant "Protein homologous cores and loops: important clues to evolutionary relationships between structurally similar proteins", BMC Struct Biol, v 7:23 (2007).</li><li class="pub_text">Benjamin A. Shoemaker and Anna R. Panchenko. "Deciphering protein-protein interactions. II. Computational methods to predict protein and domain interaction partners", PLoS Comput Biol, v. 3(3) (2007).</li><li class="pub_text">Yuri Wolf, Thomas Madej, Vladimir Babenko, Benjamin Shoemaker and Anna R. Panchenko. "Long-term trends in evolution of indels in protein sequences", BMC Evol. Biol., v.7:19 (2007).</li><li class="pub_text">Jessica Fong, Lewis Geer, Anna R. Panchenko and Stephen H. Bryant "Modeling the evolution of protein domain architectures using maximum parsimony", J. Mol. Biol., v. 366(1), p.307 (2007).</li><li class="pub_text">Saikat Chakrabarti, Christopher J. Lanczycki, Anna R. Panchenko, Teresa M. Przytycka, Paul A. Thiessen and Stephen H. Bryant "State of the art: refinement of multiple sequence alignments", BMC Bioinfomatics,14,7:499 (2006).</li><li class="pub_text">Saikat Chakrabarti, Christopher J. Lanczycki, Anna R. Panchenko, Teresa M. Przytycka, Paul A. Thiessen and Stephen H. Bryant "Refining multiple sequence alignments with conserved core regions", Nucleic Acids Res., v.34(9), p.2598 (2006).</li><li class="pub_text">Benjamin A. Shoemaker, Anna R. Panchenko and Stephen H. BryantFinding biologically relevant domain interactions: Conserved Binding Mode analysis, Protein Science, v.15(2), p.352 (2006).</li><li class="pub_text">Anna R. Panchenko, Yuri I. Wolf, Larisa Panchenko and Thomas MadejEvolutionary plasticity of protein families: coupling between sequence and structure variation, Proteins, v.61, p. 535 (2005).</li><li class="pub_text">Anna R. Panchenko and Thomas Madej Structural similarity of loops in protein families: toward the understanding of protein evolution, BMC Evolutionary Biology, 5(1), p.10 (2005).</li><li class="pub_text">Maricel Kann, Paul Thiessen, Anna R. Panchenko, Alejandro Schaffer, Stephen Altschul and Stephen BryantA Structure-based Method for Protein Sequence Alignment, Bioinformatics, v. 21(8), p.1451 (2005).</li><li class="pub_text">Anna R. Panchenko and Thomas Madej Analysis of protein homology by assessing the (dis)similarity in protein loop regions, Proteins, v.57(3), p.539(2004).</li><li class="pub_text">Anna R. Panchenko, Fyodor Kondrashov and Stephen BryantPrediction of functional sites by analysis of sequence and structure conservation, Protein Science, v.13, p. 884-892 (2004).</li><li class="pub_text">Anna R. Panchenko Finding weak protein similarities by sequence profile comparison, Nucleic Acids Res., v. 31(2), p.683-689 (2003).</li><li class="pub_text">Jie Chen, John B. Anderson, Carol DeWeese-Scott, Natalie Fedorova, Lewis Y. Geer, Siqian He, David I. Hurwitz, Cynthia A. Liebert, Chunlei Liu, Thomas Madej, Aron Marchler-Bauer, Gabriele H. Marchler, Raja Mazumder, Anastasia N. Nikolskaya, Bachoti S. Rao, Anna R. Panchenko, Benjamin A. Shoemaker, Vahan Simonyan, James S. Song, Paul A. Thiessen, Sona Vasudevan, Yanli Wang, Roxanne A. Yamashita, Jodie J. Yin and Stephen H. BryantMMDB: Entrezs 3D-structure database, Nucleic Acids Res., v. 31(1), p. 474-477 (2003).</li><li class="pub_text">Aron Marchler-Bauer, John B. Anderson, Carol DeWeese-Scott, Natalie Fedorova, Lewis Y. Geer, Siqian He, David I. Hurwitz, Cynthia A. Liebert, Chunlei Liu, Thomas Madej, Gabriele H. Marchler, Raja Mazumder, Anastasia N. Nikolskaya, Bachoti S. Rao, Anna R. Panchenko, Benjamin A. Shoemaker, Vahan Simonyan, James S. Song, Paul A. Thiessen, Sona Vasudevan, Yanli Wang, Roxanne A. Yamashita, Jodie J. Yin and Stephen H. BryantCDD: a curated Entrez database of conserved domain alignments, Nucleic Acids Res., v.31(1), p. 383-387 (2003).</li><li class="pub_text">David Lipman, Alexander Souvorov, Eugene Koonin, Anna R. Panchenko and Tatyana TatusovaThe relationship of protein conservation and sequence length, BMC Evol Biol, Nov 1;2(1):20 (2002).</li><li class="pub_text">Aron Marcheler-Bauer, Anna R. Panchenko, Naomi Ariel and Stephen BryantComparison of sequence and structure alignments for protein domains, Proteins, v.48(3), p.439-446 (2002).</li><li class="pub_text">Anna R. Panchenko and Stephen Bryant A comparison of Position-Specific Score Matrices based on sequence and structure alignments, Protein Science, v. 11, p. 361-370 (2002).</li><li class="pub_text">Yanli Wang, John B. Anderson, Jie Chen, Lewis Y. Geer, Siqian He, David I. Hurwitz, Cynthia A. Liebert, Thomas Madej, Gabriele H. Marchler, Aron Marchler-Bauer, Anna R. Panchenko, Benjamin A. Shoemaker, James S. Song, Paul A. Thiessen, Roxanne A. Yamashita, Stephen H. Bryant</li><li class="pub_text">MMDB: Entrezs 3D-Structure Database, Nucleic Acids Res., v. 30(1), p.281-283 (2002).</li><li class="pub_text">Aron Marchler-Bauer, Anna R. Panchenko, Benjamin A. Shoemaker, Paul A.Thiessen, Lewis Y. Geer and Stephen H. BryantCDD: A Database of Conserved Domain Alignments with links to domain three- dimensional structure, Nucleic Acids Res., v. 30(1), p. 249-252 (2002).</li><li class="pub_text">Anna R. Panchenko, Aron Marchler-Bauer and Stephen H. BryantCombination of threading potentials and sequence profiles improves fold- recognition, J. of Mol. Biol., v.296, p. 1319-1331 (2000).</li><li class="pub_text">Anna R. Panchenko, Aron Marchler-Bauer and Stephen H. BryantThreading with explicit models for evolutionary conservation of structure and sequence, Proteins, v. 3, p. 133-141 (1999).</li><li class="pub_text">Anna R. Panchenko, Zaida Luthey-Schulten, Roland Cole and Peter G. Wolynes</li><li class="pub_text">The foldon universe: a survey of structural similarity and self-recognition of independently folding units, J. of Mol. Biol., v. 272, p. 95-105 (1997).</li><li class="pub_text">Anna R. Panchenko, Zaida Luthey-Schulten and Peter G. WolynesFoldons as independently folding units of proteins, Physica D, v. 107, p. 312-315 (1997).</li><li class="pub_text">Anna R. Panchenko, Zaida Luthey-Schulten and Peter G. WolynesFoldons, protein structural modules and exons, Proc. Natl. Acad. Sci., v. 93, p. 2008-2013 (1996).</li><li class="pub_text">Anna R. Panchenko, Jing Wang, G. Ulrich Nienhaus and Peter G. WolynesAnalysis of ligand binding to heme proteins using a fluctuating path description, J. of Phys. Chem., v. 99, p. 9278-9282 (1995).</li><li class="pub_text">Anna R. Panchenko and K. V. Shaitan Effect of pressure on intramolecular dynamics of proteins, Molecular Biology, v. 26, p. 745-748 (1992).</li><li class="pub_text">Anna R. Panchenko and K. V. Shaitan Effect of dynamic disorder of biological systems on the characteristics of intramolecular mobility determined by Mossbauer spectroscopy, Biophysics, v. 37, p. 927-931 (1992).</li></ul></div></div></div></section><footer class="footer bg-dark"><div class="container"><div class="footer__widgets"><div class="row"><div class="col-md-4 col-sm-6 col-xs-12"><div class="widget footer__get-in-touch text-sm-center"><div class="footer__logo"></div><p class="footer__get-in-touch__paragraph"></p></div></div><div class="col-md-6 col-md-offset-2 col-sm-6 col-xs-12"><div class="widget footer__nav-links text-right text-sm-center"><ul class="list-inline"><li><a href="index.html">Home</a></li><li><a href="goal.html">Our<span class="newstitle_hide">/</span>Goal</a></li><li><a href="project_2.html">Projects</a></li><li><a href="server_1.html">Servers/Databases</a></li><li><a href="publications.html">Publications</a></li><li><a href="contact.html">Contact</a></li></ul><div class="footer__socials mt-20"><div class="social-icons social-icons--nobase"><a href="https://twitter.com/AnnaPanchenko13" target="_blank"><i class="ui-twitter"></i></a> <a href="#"><i class="ui-facebook"></i></a> <a href="#"><i class="ui-linkedin"></i></a></div></div></div></div></div></div></div></footer><div id="back-to-top"><a href="#top"><i class="ui-arrow-up"></i></a></div></div></main><script type="text/javascript" src="js/jquery.min.js"></script><script type="text/javascript" src="js/bootstrap.min.js"></script><script type="text/javascript" src="js/plugins.js"></script><script type="text/javascript" src="js/scripts.js"></script></body></html>